lib/openstudio/lib/openstudio_aws_instance.rb

```diff
@@ -95,7 +95,7 @@ def create_and_attach_volume(size, instance_id, availability_zone)
       volume_id: resp[:volume_id],
       instance_id: instance_id,
       # required
-      device: '/dev/sdh'
+      device: '/dev/sdm'
```

Comment:
is this always going to be true. I've seen sda/sdh... isn't this based on the instance type?